herring/puzzles/static/style.less

```diff
@@ -440,3 +440,9 @@ button, a.button, input[type=submit] {
 div.editable > span {
 	pointer-events: none;
 }
+
+// Possibly this should apply to all labels, but trying to be
+// ultra-conservative right now
+label[for="id_subscriptions"] {
```

Comment:
Does this literally only affect the visual appearance of the admin form? Or what is this restyling? If it's just the admin form, I don't care as long as it still works, make it pretty however you want. But I'm actually confused about why it's necessary -- the admin form appears to be styled fine to me, with labels at the top, and if I try adding this style to one of them for testing, it totally breaks the page...

Reply:
Ohhh, the admin and non-admin default forms share the exact same property for this. Amazing. I think if you apply this it might break the admin form, I'm not sure. Presumably there's some intended way to style just the user-side form...